app/models/searchgov_url.rb

```diff
@@ -27,7 +27,7 @@ class SearchgovUrl < ActiveRecord::Base
   class SearchgovUrlError < StandardError; end
 
   def fetch
-    self.last_crawled_at = Time.now
+    self.update_attributes(last_crawled_at: Time.now)
```

Comment:
This change is intended to prevent race conditions, where multiple processes looking for `unfetched` urls (`"where last_crawled_at = nil"`) might pick up the same record in the time it takes for the fetch process to run (which could be > 30 seconds).